src/Tribe/Assets.php

```diff
@@ -710,9 +702,25 @@ public function register( $origin, $slug, $file, $deps = [], $action = null, $ar
 
 		// Clean these
 		$asset->priority  = absint( $asset->priority );
-		$asset->in_footer = is_array( $asset->in_footer ) ? $asset->in_footer : (bool) $asset->in_footer; // Since WordPress 6.3, this parameter accepts an array argument.
 		$asset->media     = esc_attr( $asset->media );
 
+		// Since WordPress 6.3, the `in_footer` parameter accepts an array argument.
+		if ( version_compare( strtok( $wp_version, '-' ), '6.3', '>=' ) ) {
+			// if `in_footer` is set to boolean true, add it to the `in_footer` array. i.e. [ 'in_footer' => true ].
+			if ( (bool) $asset->in_footer ) {
```

Comment:
```suggestion
			if ( is_bool( $asset->in_footer ) ) {
```
Is this what you meant?